src/System.Device.Gpio/System/Device/Gpio/GpioDriver.cs

```diff
@@ -15,9 +15,9 @@ public abstract class GpioDriver : IDisposable
         protected internal abstract void ClosePin(int pinNumber);
         protected internal abstract void SetPinMode(int pinNumber, PinMode mode);
         protected internal abstract PinMode GetPinMode(int pinNumber);
+        protected internal abstract bool IsPinModeSupported(int pinNumber, PinMode mode);
```

Comment:
Any particular reason for the re-order?

Reply:
Not really except for topic related (e.g. PinMode, Pin read/write).  At first, I thought about alphabetical as that would be (I think) the order VS adds when implementing your own.  I can change back if needed.

Reply:
no, now that you mention it it is good to keep them topic related.